Source/CarthageKit/Xcode.swift

```diff
@@ -330,6 +352,12 @@ public enum SDK: Equatable {
 		case "iphonesimulator":
 			return .success(.iPhoneSimulator)
 
+		case "watchos":
+			return .success(.watchOS)
+
+		case "watchsimulator":
+			return .success(watchSimulator)
```

Comment:
Oh, I didn't even know you could omit the dot in this case, so that was definitely a (lucky) typo on my part. Fixing it!